benchmark/AlcorJdbcBenchmark/.idea/misc.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="ProjectRootManager" version="2" languageLevel="JDK_11" project-jdk-name="11" project-jdk-type="JavaSDK">
+    <output url="file://$PROJECT_DIR$/out" />
+  </component>
+</project>
```

Comment:
.idea files are local IDE related and should not be included.

Reply:
Intellij IDE keeps messing it up. I will remove them now.